assets/model_monitoring/components/src/feature_importance_metrics/compute_feature_importance.py

```diff
@@ -16,10 +16,7 @@
 from shared_utilities.momo_exceptions import InvalidInputError
 from shared_utilities import constants
 from sklearn.model_selection import train_test_split
-from responsibleai import RAIInsights, FeatureMetadata
-from ml_wrappers.model.predictions_wrapper import (
-    PredictionsModelWrapperClassification,
-    PredictionsModelWrapperRegression)
+from interpret_community.shap.tree_explainer import TreeExplainer
```

Comment:
I feel it is a big change, can you upgrade the component version?

Reply:
done, incremented component version